src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,104 @@
+/*eslint-disable*/
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+import About from '../pages/About';
+import NotMatch from '../pages/NotMatch';
```

Comment:
Kindly remove these lines to enable your app run successfully since those component have not yet been created or create those components.